MAINTAINERS.md

```diff
@@ -0,0 +1,50 @@
+# How to release a new version
+
+## When to release a new version
+
+New version should be released when we are ready to make changes generally available.
+
+New version should not be released if we want to test our latest changes or to make them available to a limited number of users. This can be achieved without releasing a new version. Go modules allow to point to latest master or specific Git commit.
```

Comment:
Should we also note here that if a downstream user has a bug it is not correct to simply update the downstream to a non-release commit. Release means something special to us and users of libuplink shouldn't be using non-release tags in their production code.

Reply:
It would be for testing purposes, i.e. to validate a bug fix. But, yes, it is not recommended to you non-release tags in production. Do you want to add such statement?

Reply:
Can you add the following?

```
Commits from master are *not* suitable for use in production unless they are a release tag. Do not use non-release commits in downstream projects in production.
```